### ISSUE TITLE
Crowdsource ALPR training via Mechanical Turk

### ISSUE THREAD

**Issue by matthill** [openalpr/openalpr#43]
Amazon's Mechanical Turk (https://www.mturk.com/mturk/welcome) can be used as a cost-effective way to get human operators to tag license plate images for training data.

For this to work we would need a simplified, web-based training workflow.  It would need:

  * An introductory training where users could be trained on the type of data needed
  * A validation set so that users who do not enter correct data would be disqualified
  * Training workflow for both plate regions (detection) and plate characters (OCR)
